security/qcom_fmp/README.txt

```diff
@@ -0,0 +1,38 @@
+This folder contains Firmware Management Protocol (FMP) keys for the UEFI
+Capsule Generation process [1].
+
+Details about keys generation can be found at [2].
+To generate keys the cmds below were used:
+
+The demoCA directory should be initialized:
+$ mkdir -p demoCA
+$ mkdir -p demoCA/newcerts
+$ touch demoCA/index.rst
+$ echo 01 > demoCA/serial
+
+Generate a Root Key/Certificate:
+$ openssl genrsa -aes256 -out QcFMPRoot.key 2048
+$ openssl req -new -x509 -days 3650 -key QcFMPRoot.key -out QcFMPRoot.crt
+$ openssl x509 -in QcFMPRoot.crt -out QcFMPRoot.cer -outform DER
+$ openssl x509 -inform DER -in QcFMPRoot.cer -outform PEM -out QcFMPRoot.pub.pem
+
+Generate the Intermediate Key/Certificate:
+$ openssl genrsa -aes256 -out QcFMPSub.key 2048
+$ openssl req -new -days 3650 -key QcFMPSub.key -out QcFMPSub.csr
+$ openssl ca -extensions v3_ca -in QcFMPSub.csr -days 3650 -out QcFMPSub.crt -cert QcFMPRoot.crt -keyfile QcFMPRoot.key
```

Comment:
In our `genkey` tooling we make use of a QCOM ssl config file with a param `-config opensslroot.cfg`:
https://github.com/foundriesio/qcom-sec-tools/blob/main/opensslroot.cfg

This has a setting for `demoCA` vs. the default one that I'm using in Arch Linux which has this setting:
```
dir             = /etc/ssl              # Where everything is kept
```
Because of this default setting, I end up with an error on this line:
```Using configuration from /etc/ssl/openssl.cnf
Enter pass phrase for QcFMPRoot.key:
ca: /etc/ssl/newcerts is not a directory
/etc/ssl/newcerts: No such file or directory
```